styles/style.css

```diff
@@ -1064,12 +774,8 @@ h1 a {
 
 .body h2 + p {
   -webkit-animation: glitch-in 0.4s linear 0.5s 1;
-  -moz-animation: glitch-in 0.4s linear 0.5s 1;
-  -ms-animation: glitch-in 0.4s linear 0.5s 1;
   animation: glitch-in 0.4s linear 0.5s 1;
   -webkit-animation-fill-mode: both;
-  -moz-animation-fill-mode: both;
-  -ms-animation-fill-mode: both;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The property -moz-animation is compatible with -webkit-animation and should be included as well.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1888804921/issues/source?bid=3085146&fileBranchId=3088807#l812)